temba/contacts/views.py

```diff
@@ -1464,71 +1464,27 @@ def get_gear_links(self):
             return links
 
         def get_context_data(self, *args, **kwargs):
```

Comment:
Don't we need the other keys in the context as we have here: https://github.com/greatnonprofits-nfp/rapidpro/blob/dev-2.0/temba/contacts/views.py#L1802?

Reply:
I have removed `actions` and `export_url` as they were deprecated on that page.